libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshExtra.kt

```diff
@@ -0,0 +1,53 @@
+package com.mapbox.navigation.core.routerefresh
+
+import androidx.annotation.StringDef
+import com.mapbox.api.directions.v5.models.Incident
+import com.mapbox.api.directions.v5.models.LegAnnotation
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.base.route.RouterOrigin
+import com.mapbox.navigation.core.MapboxNavigation
+
+/**
+ * Extra data of route refresh
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+object RouteRefreshExtra {
+
+    /**
+     * Whenever route refresh controller is in idle:
+     * - Trip session is not started. See [MapboxNavigation.startTripSession];
+     * - [MapboxNavigation.setNavigationRoutes] is not invoked or a list or the routes is empty;
+     * - [RouteOptions.enableRefresh] is not `true`;
+     * - Route origin of the route is [RouterOrigin.Onboard], so it cannot be refreshed from
+     * Directions API;
+     * - Refresh controller becomes idle between stages when routes are refreshed and a new round or
+     * refresh is started.
+     */
+    const val REFRESH_STATE_IDLE = "IDLE"
+
+    /**
+     * Whenever list of routes is freshened:
```

Comment:
Freshened? What exactly does it mean here? Maybe "refreshed" fits better?

Reply:
actually both mean almost the same https://wikidiff.com/refresh/freshen, but I'm fine to keep the only one word